Custom Toolbar/View/MainPage.xaml

```diff
@@ -114,6 +115,14 @@
                         Command="{Binding ZoomInCommand}" IsEnabled="{Binding CanZoomIn}"
                         PropertyChanged="Button_PropertyChanged"/>
                 </HorizontalStackLayout>
+                <Grid Grid.Column="3" Margin="0,0,12,0" HorizontalOptions="{OnPlatform MacCatalyst=End, iOS=End, Android=End, WinUI=End}">
```

Comment:
Why onplatform is required if in all the platforms is is at the end?

Reply:
I'll change sir